filament/backend/src/metal/MetalState.h

```diff
@@ -230,7 +231,8 @@ struct PipelineState {
                 this->vertexDescription == rhs.vertexDescription &&
                 std::equal(this->colorAttachmentPixelFormat, this->colorAttachmentPixelFormat + MRT::MAX_SUPPORTED_RENDER_TARGET_COUNT,
                         rhs.colorAttachmentPixelFormat) &&
-                this->depthStencilAttachmentPixelFormat == rhs.depthStencilAttachmentPixelFormat &&
+                this->depthAttachmentPixelFormat == rhs.depthAttachmentPixelFormat &&
```

Comment:
old diff, don't forget to rebase

Reply:
fixed in https://github.com/shapr3d/filament/pull/67/commits/917a91248e3bd5fdf063c55e3dcdbf473fc76d29